src/main/java/com/aws/greengrass/secretmanager/SecretManagerService.java

```diff
@@ -168,7 +142,7 @@ public void startup() throws InterruptedException {
         // We don't want to load anything if there is no file, which could happen when
         // we were not able to download any secrets due to network issues.
         try {
-            secretManager.loadSecretsFromLocalStore();
+            secretManager.reloadCache();
```

Comment:
Nit: Since the function name changed from `loadSecretsFromLocalStore` to `reloadCache`, we should update the log "// No need to log anything here, it is already logged by loadSecretsFromLocalStore" on L150 to be accurate